readme.md

```diff
@@ -48,12 +48,13 @@ theBody.style.backgroundColor = "red";
 Why does the following code snippet throw an error, and what would fix the error?
 
 ```css
-body{
+body {
   background-color = "red";
```

Comment:
Note quite. I think the space between the `body` and `{` isn't strictly necessary.

The main problem here is that we have to use a colon instead of an equal sign (and quotes around red shouldn't be there):

```css
body{
  background-color: red;
}
```